Code/Commando/WINMAIN.CPP

```diff
@@ -184,6 +184,16 @@ int main(int argc, char *argv[])
 	}
 #endif //WWDEBUG
 
+	//
```

Comment:
I'm sure there is a good reason for moving this block, its just not immediately obvious to me? Is this so FDS can do some stuff before we parse the command line and pick up our working path?